README.md

```diff
@@ -24,28 +24,52 @@ Algorand is an open-source, proof of stake blockchain and smart contract computi
 
 ## Contents
 
-- [Official Resources](#official-resources)
-- [Wallets](#wallets)
-- [Blockchain Explorers](#blockchain-explorers)
-- [Portfolio Trackers](#portfolio-trackers)
-- [Learning Resources](#learning-resources)
-  - [Crash Courses](#crash-courses)
-  - [General courses](#general-courses)
-  - [Tutorials](#tutorials)
-- [Development Tools](#development-tools)
-  - [CLI](#cli)
-  - [Languages](#languages)
-  - [IDEs](#ides)
-  - [Other Development Tools](#other-development-tools)
-- [NFT Marketplaces](#nft-marketplaces)
-- [DeFi Platforms](#defi-platforms)
-- [Security Auditing Services](#security-auditing-services)
-- [Blockchain Bridges](#blockchain-bridges)
-- [Oracles](#oracles)
-- [Name Services](#name-services)
-- [Community Resources](#community-resources)
-- [Algorand Request for Comments](#algorand-request-for-comments)
-- [Metrics and Analytics Services](#metrics-and-analytics-services)
+- [Awesome Algorand ](#awesome-algorand-)
```

Comment:
Appreciate the intent here but please remove the changes introduced to TOC. If you'd like a more detailed TOC please open an issue on the repo - ill address this in separate PR. 

Additionally - please fill the PR template according to either textual or diagram based guideline. The title of the PR will be used on a twitter post so if you wouldn't like to see `Add AlgoPoaP` as a tweet you can change it to `AlgoPoAp - {short description}` instead

Reply:
Got it thanks, changes on the way

Reply:
Oh, something is not right here, I just forked the main and made one line change and committed and PRed! Why all those detailed changes are being made! Those were not me! Checking now...